AdblockPlusSafari/AdblockPlus.m

```diff
@@ -73,53 +82,42 @@ - (void)setActivated:(BOOL)activated
   [_adblockPlusDetails synchronize];
 }
 
-#pragma mark -
-
-- (NSString *)contentBlockerIdentifier
+- (void)setFilterlists:(NSDictionary<NSString *,NSDictionary<NSString *,NSObject *> *> *)filterlists
```

Comment:
I see, fine then. Well, like I said - I'd go nuts if I have to add another language to the mix of those I need to deal with right now, and we have to use Obj-C anyway for other reasons. But in the long run we can probably move to Swift.